_scalingo_api/events.md

```diff
@@ -1771,6 +1771,114 @@ Example object:
   }
 }
 ```
+--- row ---
+
+* **Database Maintenance Planned**
+
+_When:_ A maintenance is scheduled on your database
+`type=database_maintenance_planned`
```

Comment:
The name seems wrong to me it should be `plan_database_maintenance`

Reply:
It indeed is, fixed